Automattic/woocommerce-payments
# chore: convert payment request CTA to select

Fixes https://github.com/Automattic/woocommerce-payments/issues/6470

#### Changes proposed in this Pull Request









Converting the CTA on the payment request & WooPay settings sub-pages to a "Select". See https://github.com/Automattic/woocommerce-payments/issues/6470#issuecomment-1597718155 for mockup.
I also slightly tweaked the margin-bottom below the options, to make it match the designs a little closer.

#### Testing instructions





- Go to http://wcpay.test/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments&method=woopay
- The option for the CTA should be updated
- Go to http://wcpay.test/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments&method=payment_request
- Similar behavior as above

![Screenshot 2024-02-08 at 2 52 26 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/51b027c5-4634-4a48-98b2-f64c626fc563)
![Screenshot 2024-02-08 at 2 52 22 PM](https://github.com/Automattic/woocommerce-payments/assets/273592/98a77667-8007-4442-914c-b16b5a48169a)


-------------------


**Post merge**
